docs/development.md

```diff
@@ -323,18 +323,6 @@ sed -i 's/memory: 20Mi/memory: 64Mi/' hack/install.yaml
 sed -i 's/memory: 30Mi/memory: 128Mi/' hack/install.yaml
 ```
 
-Then, delete the `type` field from the OpenAPI schema to avoid triggering validation
```

Comment:
Please remove this fix from `scripts/generate-install-yaml.sh` as well.

Reply:
Done